daemon/daemon.go

```diff
@@ -1410,6 +1410,10 @@ func (daemon *Daemon) networkOptions(dconfig *config.Config, pg plugingetter.Plu
 
 	options = append(options, nwconfig.OptionNetworkControlPlaneMTU(dconfig.NetworkControlPlaneMTU))
 
+	if len(dconfig.NetworkConfig.HostServiceDynamicPortRange) > 0 {
```

Comment:
I guess we should have some validation, as we likely want this to be within the regular ephemeral port range (ie, don't want port 80 to be wittin this range); also validate that the value is < 65535 (likely lower, to have a reasonable _range_)

Reply:
the validation is in libnetwork